### ISSUE TITLE
`subset_shape` fails if dataset and shape have no CRS

### ISSUE THREAD

**Issue by huard** [roocs/clisops#95]
* clisops version: master

### Description
`subset_shape` fails at `shape_crs = CRS(poly.crs)` if `poly.crs` is `None`, which is the case if shape is a simple geometry with no CRS information attached.

**Comment by Zeitsperre** [comment]
I can look into this. All the parts are there. It can probably be fixed with a try-except catch.

**Comment by huard** [comment]
Ok thanks,

I'd like a simple test like this to work: 
```python
from shapely.geometry import Polygon
# Create dummy Dataset
dims = ("lon", "lat")
n = {"lon": 5, "lat": 6}
ds = xr.Dataset(data_vars={"foo": (dims, np.ones(shape=list(n.values())))},
                coords={"lon": ("lon", np.arange(n["lon"])),
                        "lat": ("lat", np.arange(n["lat"]))}
                )
poly = Polygon([[0.5, 0], [2.5, 0], [2.5, 2.5], [0.5, 2.5]])
shape = gpd.GeoDataFrame(geometry=[poly])
subset.subset_shape(ds, shape=shape)
```

**Comment by Zeitsperre** [comment]
fixed with #96
